app/components/c-collapse.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+
+  <div :class="b({expanded: isExpanded})">
+    <div :class="b('toggle')" @click="setState">
+      <e-icon :class="b('icon')" :inline="true" icon="i-plus"/>
+      {{ title }}
+    </div>
+
+    <div :class="b('content')">
+      <div :class="b('inner')">
+        <!-- @slot Used for item content -->
+        <slot></slot>
+      </div>
+    </div>
+
+  </div>
+
+</template>
+
+<script>
+  import EventBus from '@/setup/event-bus';
+
+  /**
+   * Displays collapsible content panels. Use **c-collapse-group** as a wrapper for multiple items
+   * and if only one item should be open at the same time (:one-active=true).
+   */
+  export default {
+    name: 'c-collapse',
+    // components: {},
+    // mixins: [],
+
+    props: {
+      /**
+       * Sets state of the item
+       */
+      active: {
+        type: Boolean,
+        required: false,
```

Comment:
Do you need here a required: false? Isn't it per default on false? Specially if you define a default value I think it's not necessary